pkg/apis/nfs.rook.io/v1alpha1/types.go

```diff
@@ -0,0 +1,95 @@
+/*
```

Comment:
did you try running `make codegen` yet? It will generate code around these new types.